### PR TITLE
refactor: (Style) changed px to rem

### PR DESCRIPTION
**Pull request by Aditi0505**
# Description
- Changed px values to rem
- Visually nothing changed hence, not attaching the screenshots.

# Deploy link
https://deploy-preview-17--redazzleui.netlify.app/

### CODE REVIEW

File: components/basic.css [hunk 1]
```diff
@@ -57,6 +57,9 @@
   text-decoration: none;
   box-sizing: border-box;
 }
+html {
+  font-size: 62.5%;
+}
 .padding-xs {
```
Comment (rohanmathur91) [inline components/basic.css:63]: use any tool/extension for proper spacing between lines.

**Review by rohanmathur91 — APPROVED** [review]
take care of spacing between classes, rest looks good.

File: components/basic.css (hunk 1 above)
Comment (Aditi0505) [reply components/basic.css:63]: I have been using prettier. Any other tool suggestion?

**Review by surbhi84 — APPROVED** [review]
looks good to me.

**Review by manojncsoni007 — COMMENTED** [review]
looks good

**Pull request merged**